app/extensions/safe/test/e2e/safe-auth.spec.js

```diff
@@ -179,6 +182,39 @@ describe( 'safe authenticator protocol', () =>
         } );
     }
 
+    it.only( 'can login, log history, logout, and clean up history', async ( ) =>
```

Comment:
A lingering `.only` @hunterlester

Reply:
@joshuef Oh right, thank you. I didn't mean to push to this branch. I'm experimenting with this e2e test.